web/libs/storybook/.storybook/main.ts

```diff
@@ -0,0 +1,34 @@
+import type { StorybookConfig } from '@storybook/react-vite';
+
+import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin';
+import { mergeConfig } from 'vite';
+import svgr from 'vite-plugin-svgr';
+
+const config: StorybookConfig = {
+  stories: [
+    '../src/lib/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+    '../src/stories/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+  ],
+  addons: ['@storybook/addon-essentials', '@storybook/addon-interactions'],
+  framework: {
+    name: '@storybook/react-vite',
+    options: {},
+  },
+
+  viteFinal: async (config) =>
+    mergeConfig(config, {
+      plugins: [
+        nxViteTsPaths(),
+        svgr({
+          esbuildOptions: {},
+          svgrOptions: { plugins: ['@svgr/plugin-jsx'], ref: true },
```

Comment:
Not sure how does this work exactly, but in all our other configs we also use `removeViewBox: false` for proper work of some icons.

Reply:
just checked, it doesn't remove it, so we're good

Reply:
ah, yes, NX does this automatically, that's why it was good in monorepo and failed during back-porting!